pom.xml

```diff
@@ -5343,18 +5351,21 @@
         <scope>test</scope>
       </dependency>
       
-      <!-- Used by jbpm-workitems-rest -->
+      <!-- Used by jgit -->
       <dependency>
         <groupId>org.bouncycastle</groupId>
         <artifactId>bcprov-jdk15on</artifactId>
         <version>${version.org.bouncycastle}</version>
-        <scope>test</scope>
       </dependency>
```

Comment:
this is a little bit controversial change as it was only test dependency as noted above and beside that excluded from transitive chains of our deps.

Reply:
do we really need to manage it and do not exclude it from jgit dependency?

Reply:
`test` scope set for `bouncycastle` dependencies in https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1488/commits/e6bdce2971bf3ba12dcb5e5c1486b343b2f52362.